.github/workflows/docusaurus.yml

```diff
@@ -0,0 +1,8 @@
+name: docusaurus
```

Comment:
@qwaxys Please use a separate PR as this is not related to the subject of this one.